src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -10,10 +10,27 @@
     <CodeAnalysisRuleSet>..\..\FakeItEasy.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup Label="Package properties">
+    <Title>FakeItEasy - Mocking made faking easy!</Title>
+    <Authors>Patrik Hägne, FakeItEasy contributors</Authors>
+    <Description>It's faking amazing! The easy mocking framework for .NET that works great in C# and VB.NET alike. No need to know the difference between a stub, a mock or a spy, everything's a fake! The easy to use, refactoring friendly API makes faking a breeze.</Description>
+    <Copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</Copyright>
+    <PackageProjectUrl>http://fakeiteasy.github.io/</PackageProjectUrl>
+    <PackageIconUrl>http://fakeiteasy.github.io/img/fakeiteasy_logo_256_square_white.png</PackageIconUrl>
```

Comment:
outside the scope of the issue, but while you're moving the URLs, we could make them HTTPS…

I will not hold up the mege for this